### ISSUE TITLE
generate:entity:content base-path for the content entity routes does not work

### ISSUE THREAD

**Issue by liquidcms** [hechoendrupal/DrupalConsole#2974]
At least from the interactive prompt when i enter a value of /admin/structure/reservation for the base-path for the content entity routes, i still get the routes being at admin/structure.

this leads to:
- /admin/structure/Reservation settings
- /admin/structure/Reservation list

unless, this is due to my entering the path as simply /admin/structure the first time and DC not being able to update when entity is recreated.. which i guess would be a different bug.

**Comment by liquidcms** [comment]
hmm... no, it seems as though DC simply can't edit/delete anything that has been added to a custom module. I think (so far) only option to edit an entity added to a module is to delete the entire module and start from scratch.

**Comment by liquidcms** [comment]
Nope, even if i redo module from scratch and enter base path at command line; it still hard codes it to the wrong place

**Comment by jmolivas** [comment]
@liquidcms You are right, code generation only creates files 

Can you clarify what is the proper path to work on a PR to have this issue fixed.

**Comment by liquidcms** [comment]
good question and i think it is going to be different for each case.

but in general i don't think you want to clutter up items at the main Drupal admin parents such as admin/structure, etc. 

also, main point of my post was i don't think it was possible to set the routing path with the option.

as it turned out i was creating 2 entities for my module; which give 4 menu items. Ideally they shouldnt all be at admin/structure. this doesnt seem to be correct.. but best i could come up with:

main entity is a reservation and secondary entity is a reservation category, so i went with: 

- admin/structure/reservations    -> reservations list
- admin/structure/reservations/settings    -> reservations settings (which may or not exist)
- admin/structure/reservations/categories    -> reservation cats list
- admin/structure/reservations/categories/settings    -> reservations cats settings

and i can only set this up by editing the [module].links.menu.yml - which was easy and worked well.

**Comment by Mithenks** [comment]
Hello,
I would like to report more information on this bug based on my experience.

I'm trying to create a new Content Entity using the command **generate:entity:content**. I want that the routes for this content entity have as base-path something different from the default _/admin/structure_; say, for example, that I want to put these routes in _/admin/config_ path.

You can find the output of the drupal console command in
 [drupal_console_issue_2974_log_1.txt](https://github.com/hechoendrupal/drupal-console/files/1252017/drupal_console_issue_2974_log_1.txt).

After this, the content entity routes are placed under _/admin/structure_.  
If I look at the file _modules/custom/test_module/test_module.links.menu.yml_ generated by the drupal console command, I notice that the parent parameter is not set according to what I have specified (see [drupal_console_issue_2974_log_2.txt](https://github.com/hechoendrupal/drupal-console/files/1252018/drupal_console_issue_2974_log_2.txt))

Given a quick look at the source code, I have noticed that the template used to generate this file has the _parent_ line hard-coded (see: [links.menu-entity-content.yml.twig](https://github.com/hechoendrupal/drupal-console/blob/master/templates/module/links.menu-entity-content.yml.twig)) 

So, to solve this, I think we need to:
1. Calculate the route in which put the content entity links from the config path specified by the user (see: line 89 of [EntityContentGenerator.php](https://github.com/hechoendrupal/drupal-console/blob/master/src/Generator/EntityContentGenerator.php)) 
2. Pass this value to the twig template and write down to the generated file


Hope this helps.

Regards, 
  Michele

**Comment by harold20** [comment]
The content generated by generate:entity:content command will be change at Drupal Console 2.0
